integration-tests/tests/api/artifacts-cdn.spec.ts

```diff
@@ -404,7 +404,7 @@ function runArtifactsCDNTests(
             'content-type': 'application/json',
           },
           body: JSON.stringify({
-            query: /* GraphQL */ `
```

Comment:
Why remove this? It helps prettier with formatting

Reply:
To remove warning, since it’s anonymous query